OpenSim/Simulation/MarkersReference.h

```diff
@@ -152,32 +113,42 @@ OpenSim_DECLARE_CONCRETE_OBJECT(MarkersReference, Reference_<SimTK::Vec3>);
     /** get the names of the markers serving as references */
     const SimTK::Array_<std::string>& getNames() const override;
     /** get the value of the MarkersReference */
-    void getValues(const SimTK::State &s, SimTK::Array_<SimTK::Vec3> &values) const override;
+    void getValues(const SimTK::State &s,
+        SimTK::Array_<SimTK::Vec3> &values) const override;
     /** get the speed value of the MarkersReference */
-    virtual void getSpeedValues(const SimTK::State &s, SimTK::Array_<SimTK::Vec3> &speedValues) const;
+    virtual void getSpeedValues(const SimTK::State &s,
+        SimTK::Array_<SimTK::Vec3> &speedValues) const;
     /** get the acceleration value of the MarkersReference */
-    virtual void getAccelerationValues(const SimTK::State &s, SimTK::Array_<SimTK::Vec3> &accValues) const;
-    /** get the weighting (importance) of meeting this MarkersReference in the same order as names*/
-    void getWeights(const SimTK::State &s, SimTK::Array_<double> &weights) const override;
+    virtual void getAccelerationValues(const SimTK::State &s,
+        SimTK::Array_<SimTK::Vec3> &accValues) const;
+    /** get the weighting (importance) of meeting this MarkersReference in the
+        same order as names*/
+    void getWeights(const SimTK::State &s,
+                    SimTK::Array_<double> &weights) const override;
 
     //--------------------------------------------------------------------------
     // Convenience Access
     //--------------------------------------------------------------------------
     double getSamplingFrequency() {return _markerData->getDataRate(); }
-    Set<MarkerWeight> &updMarkerWeightSet() {return _markerWeightSet; }
-    void setMarkerWeightSet(Set<MarkerWeight> &markerWeights);
-    void setDefaultWeight(double weight) {_defaultWeight = weight; }
+    Set<MarkerWeight> &updMarkerWeightSet() {return upd_marker_weights(); }
+    void setMarkerWeightSet(const Set<MarkerWeight>& markerWeights);
```

Comment:
What do you think about a doxygen comment saying that starting in 4.0 this method now makes a copy (instead of referencing the objects in the passed-in set) in case people previously had code where they continued to modify their set, thinking it affected the MarkersReference?